featurebyte/common/progress.py

```diff
@@ -0,0 +1,38 @@
+"""
+Utilities related to progress update
+"""
+from __future__ import annotations
+
+from typing import Callable
+
+
+def set_progress_range(
```

Comment:
Would `get_ranged_progress_callback` be more descriptive of what the function does?

Reply:
Indeed, will rename.